va/va.go

```diff
@@ -110,6 +119,12 @@ func New(
 		va.log.Printf("Disabling random VA sleeps")
 	}
 
+	sleepTime := os.Getenv(sleepTimeEnvVar)
+	sleepTimeInt, err := strconv.Atoi(sleepTime)
+	if err == nil && sleepTimeInt >= 1 {
+		va.sleepTime = sleepTimeInt
```

Comment:
Can you add a `va.log.Printf` that outputs the `va.sleepTime` when it is overridden?

Reply:
Sure.